global/supportedLanguages.json

```diff
@@ -248,8 +247,7 @@
     "store": "Magasin",
     "submit": "Soumettre",
     "sun": "Soleil",
-    "temperature": "Température (°C)",
-    "dateExceeds8Days": "*Impossible d'obtenir la météo pour les dates à plus de 8 jours*",
```

Comment:
Shouldn't be removed.

Reply:
I can still see these lines in this file